sites/partners/src/components/listings/PaperListingForm/index.tsx

```diff
@@ -444,7 +444,7 @@ const ListingForm = ({ listing, editMode }: ListingFormProps) => {
             size="sm"
           >
             {t("listings.actions.close")}
-          </Button>,
+          </Button>
```

Comment:
This is existing, but just recording it here that I don't think these are the right color buttons